### PR TITLE
(feature) Bootstrapify Games page and two other pages relying on its styles

### PR DESCRIPTION
**Pull request by developerrowan**

### CODE REVIEW

File: app/games/all-games-card-header.component.tsx [hunk 1]
```diff
@@ -15,17 +14,13 @@ export const AllGamesCardHeader: React.FunctionComponent<
 > = ({ game }) => {
     const gameUrl = getGameUrl(game);
     return (
-        <Card.Header className={styles.cardHeader}>
-            <div style={{ overflow: "hidden" }}>
-                <a href={`/games/${gameUrl}`} style={{ fontSize: "large" }}>
+        <Card.Header>
```
Comment (zoglo) [inline app/games/all-games-card-header.component.tsx:17]: You might want to remove the border from the card header by attaching className="border-0" to it

File: app/games/all-games-card.component.tsx [hunk 2]
```diff
@@ -16,7 +16,13 @@ export const AllGamesCard: React.FunctionComponent<AllGamesCardProps> = ({
     return (
         <Col xl={6} lg={12} key={game.game}>
             <AllGamesImage game={game} />
-            <Card className="card-columns">
+            <Card
+                className="card-columns text-nowrap"
+                style={{
```
Comment (zoglo) [inline app/games/all-games-card.component.tsx:21]: Is that inline style really necessary?

As these are being used a few times, I'd say to actually move that right into globals (for now) and attach a class like "game-card" to it

File: app/games/all-games-image.component.tsx [hunk 3]
```diff
@@ -17,24 +16,29 @@ export const AllGamesImage: React.FunctionComponent<AllGamesImageProps> = ({
...
     return (
-        <div className={styles.image}>
+        <div
+            className="float-start d-flex d-none d-sm-block align-items-center min-h-152p me-2"
+            style={{ minWidth: "100px" }}
+        >
             <a href={`/games/${gameUrl}`}>
                 {game.image && game.image != "noimage" && (
                     <GameImage
+                        className="min-h-152p"
```
Comment (zoglo) [inline app/games/all-games-image.component.tsx:26]: As the height is already set, you don't need the min-h-152p custom class you've added.
However, you could add the img-fluid class (https://getbootstrap.com/docs/5.3/content/images/#responsive-images)

File: app/tournaments/all-tournaments.tsx [hunk 4]
```diff
@@ -49,12 +48,10 @@ export const ListTournaments = ({
...
                     >
                         {tournament.logoUrl && (
-                            <div
-                                className={styles.image}
-                                style={{ marginLeft: "0.5rem" }}
-                            >
+                            <div className="float-start d-flex d-none d-sm-block align-items-center min-h-152p me-2 ms-2">
                                 <a href={`/tournaments/${tournament.name}`}>
                                     <Image
+                                        className="min-h-152p"
```
Comment (zoglo) [inline app/tournaments/all-tournaments.tsx:54]: Same here with the img-fluid class

File: src/styles/_globals.scss [hunk 5]
```diff
@@ -158,6 +158,10 @@ body {
   list-style-position: inside;
 }
 
+.min-h-152p {
+  min-height: 152px;
```
Comment (zoglo) [inline src/styles/_globals.scss:162]: You'll most likely remove this with the static heights and the img-fluid class.

File: src/styles/bootstrap/_utilities.scss [hunk 6]
```diff
@@ -58,6 +59,7 @@ $utilities: map-merge(
                   90:   90%,
                   55p:  55px,
                   110p: 110px,
+                  152p: 152px,
```
Comment (zoglo) [inline src/styles/bootstrap/_utilities.scss:62]: Same here with the img-fluid class.

**Review by zoglo — COMMENTED** [review]
As seen in the comments above

File: app/games/all-games-image.component.tsx (hunk 3 above)
Comment (developerrowan) [reply app/games/all-games-image.component.tsx:26]: The issue is with img-fluid (which I tried prior the PR), it sets the `height` to `auto` causing the image to be squished and hanging from the top of the container. Explicitly setting the min-height is the only way I got this to work...

File: src/styles/bootstrap/_utilities.scss (hunk 6 above)
Comment (developerrowan) [reply src/styles/bootstrap/_utilities.scss]: this was mistakenly left behind, it's gone now

**Pull request merged**